hack/prepare-for-integration-tests.sh

```diff
@@ -136,7 +136,7 @@ if ! tilt_mode; then
   tag=$(uuidgen) # always a new tag to force K8s to reload the image on redeploy
 
   if [[ "$skip_build" == "yes" ]]; then
-    most_recent_tag=$(docker images "$repo" --format "{{.Tag}}" | head -1)
+    most_recent_tag=$(docker images "$registry/$repo" --format "{{.Tag}}" | head -1)
```

Comment:
@cfryanr - I hope this will fix that `--skip-build` regression that we talked about briefly the other day.